safe_landing_planner/include/safe_landing_planner/grid.hpp

```diff
@@ -0,0 +1,79 @@
+#pragma once
+
+#include <Eigen/Core>
+#include <Eigen/Dense>
+
+namespace avoidance {
+
+class Grid {
+public:
+  Grid(const float grid_size, const float cell_size) :
+    grid_size_(grid_size),
+    cell_size_(cell_size),
+    mean_(static_cast<int>(std::ceil(grid_size/cell_size)), static_cast<int>(std::ceil(grid_size/cell_size))),
```

Comment:
I think make a member variable which captures the result of `static_cast<int>(std::ceil(grid_size/cell_size))`, then just use that. Something like `grid_cells_`